packages/angular_devkit/build_angular/src/babel/webpack-loader.ts

```diff
@@ -136,14 +136,11 @@ export default custom<ApplicationPresetOptions>(() => {
         // During the transition, this will always attempt to load the entry point for each file.
         // This will only occur during prerelease and will be automatically corrected once the new
         // entry point exists.
-        // TODO_ESM: Make import failure an error once the `tools` entry point exists.
         if (i18nPluginCreators === undefined) {
           // Load ESM `@angular/localize/tools` using the TypeScript dynamic import workaround.
           // Once TypeScript provides support for keeping the dynamic import this workaround can be
           // changed to a direct dynamic import.
-          try {
-            i18nPluginCreators = await loadEsmModule<I18nPluginCreators>('@angular/localize/tools');
-          } catch {}
+          i18nPluginCreators = await loadEsmModule<I18nPluginCreators>('@angular/localize/tools');
```

Comment:
Nit: Can we drop the if-statement now and simplify to `i18nPluginCreaters ??= await loadEsmModule(/* ... */);`?

Reply:
Good catch